packages/scaffold-ui/src/partials/w3m-connect-injected-widget/index.ts

```diff
@@ -53,7 +54,7 @@ export class W3mConnectInjectedWidget extends LitElement {
             return null
           }
 
-          if (!ConnectionController.checkInstalled()) {
+          if (!ConnectionController.checkInstalled(undefined, connector.chain)) {
```

Comment:
Why passing undefined?

Reply:
Okay previously it has been called like the following, not to pass the chain param, needed to pass undefined for empty first param.
```js
ConnectionController.checkInstalled()
```

Reply:
If there is an optional parameter here, we should put it last